images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile.yml

```diff
@@ -7,12 +7,11 @@
     msg: "Unsupported spec.storage.type, only 'hive' is  a supported option"
 
 - name: Validate Hive storage configuration
-  assert:
-    that:
-      - hiveStorageType is not undefined and hiveStorageType in ['s3', 'sharedPVC', 'hdfs', 'azure']
-    msg: "Invalid spec.storage.hive.type: '{{ hiveStorageType }}', must be one of 's3', 'azure', or 'sharedPVC'"
+  fail:
+    msg: "Invalid spec.storage.hive.type: '{{ hiveStorageType }}', must be one of 's3', 'azure', 'gcs', or 'sharedPVC'"
```

Comment:
This looks like a incorrect merge conflict resolution. We switched to using `assert`, so this change needs to be reverted back and updated to use assert.